src/server/editorServices.ts

```diff
@@ -509,7 +509,7 @@ export function convertUserPreferences(preferences: protocol.UserPreferences): U
     return userPreferences;
 }
 
-export interface HostConfiguration {
+interface HostConfiguration {

```

Comment:
This looks like it may be used: https://github.com/search?q=%2F%5Cbts%5C.server%5C.HostConfiguration%2F+%28language%3ATypeScript+OR+language%3AJavaScript%29&type=code